src/Drivers/nlpSparse_ex6_driver.cpp

```diff
@@ -176,6 +176,7 @@ int main(int argc, char **argv)
     nlp.options->SetStringValue("linear_solver_sparse", "pardiso");
   }
   if(use_cusolver) {
+    nlp.options->SetStringValue("linsol_mode", "speculative");
     nlp.options->SetStringValue("linear_solver_sparse", "cusolver");
     nlp.options->SetStringValue("compute_mode", "hybrid");
```

Comment:
Need to enable `inertia_free` for cuSolver:
```suggestion
    nlp.options->SetStringValue("linsol_mode", "speculative");
    nlp.options->SetStringValue("linear_solver_sparse", "cusolver");
    nlp.options->SetStringValue("compute_mode", "hybrid");
    nlp.options->SetStringValue("fact_acceptor", "inertia_free");
```

Reply:
Or merge with `cusolver-inertia-free-dev` :)

Reply:
> Or merge with `cusolver-inertia-free-dev` :)

I hacked it for now
https://github.com/LLNL/hiop/blob/22ad66217e79c3d4086cc2c79e0cf908e722c681/src/Optimization/hiopKKTLinSysSparse.cpp#L638

Reply:
No, the issue is with sparse ex6, not "fake inertia".

Reply:
it does not work with 
`
nlp.options->SetStringValue("fact_acceptor", "inertia_free");
`

Reply:
I can run it on my machines without calls to fake inertia, see #440.